libvuln/driver/driver.go

```diff
@@ -73,3 +76,20 @@ type Fetcher interface {
 	// a sha265 sum of a OVAL xml file.
 	Fetch() (io.ReadCloser, string, error)
 }
+
+// FetcherNG is an experimental fetcher interface.
+//
+// This may go away or be renamed without warning.
+type FetcherNG interface {
```

Comment:
Do we want to make this the interface here-on-out ?

Reply:
Maybe. Keeping it separate for a bit to feel it out means we don't have to port all the code over at once while we find something that works.

Reply:
:+1: